Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -326,7 +310,7 @@ void doPublishOdom() {
   double vth = (vr - vl)/(2 * TRACK_RADIUS);
   
   // Calculate the deltas since
-  double dt = current_time.toSec() - last_odom_publish_time.toSec();
+  double dt = current_time.toSec() - last_raw_odom_publish_time.toSec();
```

Comment:
At home, I don't even use the IDE. I use vim to edit and use the arduino cli to compile and upload. The ide annoys me.